notebooks/gnn/mag240m_pg.ipynb

```diff
@@ -0,0 +1,405 @@
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Feature variable\n",
+    "skip_edges = False\n",
+    "skip_features = True\n",
+    "\n",
+    "load_paper_features = True\n",
+    "load_paper_labels = True\n",
+    "load_paper_year = True\n",
+    "\n",
+    "load_affiliation_edges = False\n",
+    "load_writes_edges = True\n",
+    "load_cites_edges = False"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import needed libraries. \n",
+    "# We recommend using the [cugraph_dev](https://github.com/rapidsai/cugraph/tree/branch-22.10/conda/environments) env through conda\n",
+    "\n",
+    "from dask.distributed import Client, wait\n",
+    "from dask_cuda import LocalCUDACluster\n",
+    "from cugraph.dask.comms import comms as Comms\n",
+    "import cugraph.dask as dask_cugraph\n",
+    "import cugraph\n",
+    "import dask_cudf\n",
+    "import cudf\n",
+    "import time\n",
+    "import numpy as np\n",
+    "import math"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setup variables"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# size of dataset\n",
+    "class Dataset:\n",
+    "    def __init__(self):\n",
+    "        self.num_papers  = 121751666\n",
+    "        self.num_authors = 122383112\n",
+    "        self.num_institutions = 25721\n",
+    "        self.num_paper_features = 768\n",
+    "        self.num_classes = 153\n",
+    "        self.num_cite_edges = 1297748926\n",
+    "        self.num_write_edges = 386022720\n",
+    "        self.num_affiliated_edges = 44592586\n",
+    "    \n",
+    "    def adjust_by(self, factor):\n",
+    "        self.num_papers = math.floor(self.num_papers * factor)\n",
+    "        self.num_authors = math.floor(self.num_authors * factor)\n",
+    "        self.num_institutions = math.floor(self.num_institutions * factor)\n",
+    "        self.num_paper_features = math.floor(self.num_paper_features * factor)\n",
+    "        self.num_classes = math.floor(self.num_classes * factor)\n",
+    "        self.num_cite_edges = math.floor(self.num_cite_edges * factor)\n",
+    "        self.num_write_edges = math.floor(self.num_write_edges * factor)\n",
+    "        self.num_affiliated_edges = math.floor(self.num_affiliated_edges * factor)        \n",
+    "\n",
+    "dataset = Dataset()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Directories and Files"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mag_dir = \"/home/brad/Notebooks/dataset/mag240m_kddcup2021/processed\"\n",
```

Comment:
Was this path meant to be in this PR?  Is it supposed to be based off of the `base_dir` variable used in the example above?
```
dataset = MAG240MDataset(root = base_dir)
```
Also, you might consider using `pathlib` instead of piecing larger paths together using string `+` operators:
```
from pathlib import Path

mag_dir = Path("...")
paper_feature_file = mag_dir / "paper/node_feat.npy"
...
```
which eliminates errors due to misplaced leading "/".

Reply:
oops.   I'll clean that up